hue/hue/base/kustomization.yaml

```diff
@@ -0,0 +1,108 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - hive-site-xml-secret.yaml
+  - hue-db-init-job.yaml
```

Comment:
This file is missing from the commit

Reply:
I've removed it and forgot it in the kustomization, thank you! :smile: 

https://github.com/opendatahub-io/odh-manifests/pull/227#discussion_r536092305